apps/ledger-live-mobile/src/screens/Portfolio/index.tsx

```diff
@@ -256,15 +254,7 @@ function PortfolioScreen({ navigation }: Props) {
         : []),
       ...(showAssets
         ? [
-            <Box background={colors.background.main}>
```

Comment:
Shouldn't we keep the same format as before ?
SectionTitle outside of the section.
And, like i see on the screen, if the section title css have changed, just change it to the new one ? 

Something like this : 

`
 <Box>
<SectionContainer px={0} mb={8}>
                <SectionTitle
                  title={t("analytics.operations.title")}
                  containerProps={{ mx: 6 }}
                />
                <OperationsHistorySection accounts={assetsToDisplay} />,
              </SectionContainer>
            </Box>,
`